### ISSUE TITLE
Example failure

### ISSUE THREAD

**Issue by bwtakacy** [embulk/embulk#154]
Hi,

Thanks for interesting tool!

I tried to execute the command noted in 'Trying the example' on README.md, but encountered following error.

```
$ embulk preview config.yml
2015-04-09 14:10:27.421 +0900: Embulk v0.5.5
org.embulk.config.ConfigException: Attribute type is required but not set
        at org.embulk.config.DataSourceImpl.get(org/embulk/config/DataSourceImpl.java:73)
        at org.embulk.spi.FileInputRunner.newParserPlugin(org/embulk/spi/FileInputRunner.java:56)
        at org.embulk.spi.FileInputRunner$RunnerControl.<init>(org/embulk/spi/FileInputRunner.java:100)
        at org.embulk.spi.FileInputRunner.transaction(org/embulk/spi/FileInputRunner.java:63)
        at org.embulk.exec.PreviewExecutor.doPreview(org/embulk/exec/PreviewExecutor.java:93)
        at org.embulk.exec.PreviewExecutor.access$000(org/embulk/exec/PreviewExecutor.java:27)
        at org.embulk.exec.PreviewExecutor$1.run(org/embulk/exec/PreviewExecutor.java:67)
        at org.embulk.exec.PreviewExecutor$1.run(org/embulk/exec/PreviewExecutor.java:63)
        at org.embulk.spi.Exec.doWith(org/embulk/spi/Exec.java:22)
        at org.embulk.exec.PreviewExecutor.preview(org/embulk/exec/PreviewExecutor.java:63)
        at org.embulk.command.Runner.preview(org/embulk/command/Runner.java:240)
        at org.embulk.command.Runner.main(org/embulk/command/Runner.java:100)
        at java.lang.reflect.Method.invoke(java/lang/reflect/Method.java:606)
        at RUBY.run(file:/home/postgres/.embulk/bin/embulk!/embulk/command/embulk_run.rb:299)
        at classpath_3a_embulk.command.embulk.(root)(classpath:embulk/command/embulk.rb:43)
        at classpath_3a_embulk.command.embulk.(root)(classpath_3a_embulk/command/classpath:embulk/command/embulk.rb:43)
        at org.embulk.cli.Main.main(org/embulk/cli/Main.java:13)

Error: Attribute type is required but not set
```

I succeeded  to run example with Embulk v0.5.4. 
The output config.yml from guess command seems to be a little different from v0.5.5, but I could not find how to solve this...

Regards,

**Comment by frsyuki** [comment]
That's a serious bug! I fixed it. I'm now working on releasing a hotfix.

**Comment by frsyuki** [comment]
I fixed and released v0.6.1. Thank you for your reporting!
